GitVersion.yml

```diff
@@ -1,7 +1,7 @@
 mode: ContinuousDeployment
 branches:
   master:
-    regex: ^master
+    regex: ^master|main$
```

Comment:
From reading the docs, this is the default (changed by this PR - https://github.com/GitTools/GitVersion/pull/2367) so it looks like we don't need this line at all. Happy for you to leave it in though, if you prefer it to be explicit.

Reply:
I had seen that, but https://github.com/GitTools/GitVersion/issues/2433 remains open and people are reporting still seeing problems even after the change, so I wasn't convinced they were done yet. Nor could I tell which (if any) version of GitVersion it appears in, and we're pinned to 5.6.6 at the moment, so I couldn't be sure we are picking up the fix. So that's why I'm doing it this way.